dmotion:
  ankle_yaw:
    data: [0, 0.94035934630056972, 1.9802164180260569, 2.3053189269978605, 3.1606752800111555, 4.3056869024631466, 5.5, 0, -6.9216393290732974, -6.9050195057907899, 15.023294043165082, -0.059094986559243529, -0.0071727338715100748, 0, 0, 0.029679307131801114, -0.052407779283043139, 4.7453400774557997, 0.01815341795882541, 0, 0]